client/app/services/intercom.js

```diff
@@ -1,13 +1,18 @@
 import Ember from 'ember';
 import config from 'client/config/environment';
 
+/* eslint-disable new-cap*/
 export default Ember.Service.extend({
-  intercom: window.Intercom || Ember.K,
+  init() {
+    this._super(...arguments);
+    if (config.intercom.disabled) {
+      window.Intercom = Ember.K;
```

Comment:
I'm confused about the overall changes to this service. Last I booted the app in development/test, everything was working fine and not throwing errors. Is that no longer the case?